test/keychain.js

```diff
@@ -0,0 +1,45 @@
+const test = require('brittle')
+const { swarm } = require('./helpers')
+const Keychain = require('keypear')
+
+test('createServer with keypear + connect', async function (t) {
```

Comment:
It's the same test as in `test/connections.js`, except that in `server.listen()` uses `keys.get()` from keypear instance.